README.md

```diff
@@ -476,75 +451,19 @@ Events:
 
 Liveness and Readiness probe failures indicate the pod is taking longer than expected to come alive/online, check pod logs.
 
-It might also be useful to transfer all logs from a pod to a directory on the host for further examination, we can use _oc rsync_ for this task.
-
-```bash
-$ oc rsync <pod-name>:/var/www/miq/vmdb/log ./pod-logs/
-receiving incremental file list
-log/
-log/.gitkeep
-log/api.log
-log/audit.log
-log/automation.log
-log/aws.log
-log/azure.log
-log/container_monitoring.log
-log/datawarehouse.log
-log/evm.log
-log/fog.log
-log/gem_list.txt
-log/kubernetes.log
-log/last_settings.txt
-log/last_startup.txt
-log/lenovo.log
-log/package_list_rpm.txt
-log/policy.log
-log/production.log
-log/rhevm.log
-log/scvmm.log
-log/vcloud.log
-log/vim.log
-log/websocket.log
-
-sent 452 bytes  received 1,247,768 bytes  832,146.67 bytes/sec
-total size is 1,246,124  speedup is 1.00
-```
-
 ## Building Images on OpenShift
-It is possible to build the images from this repository (or any of other) using OpenShift:
-
-```bash
-$ oc -n <your-namespace> new-build --context-dir=images/<image-directory> https://github.com/ManageIQ/manageiq-pods#master
-```
-
-In addition it is also suggested to tweak the following `dockerStrategy` parameters to ensure fresh builds every time:
+It is possible to build the images used for this project in OpenShift.
 
-```bash
-$ oc edit bc -n <your-namespace> manageiq-pods
-```
-
-```yaml
-strategy:
-  dockerStrategy:
-    forcePull: true
-    noCache: true
-```
+A template is provided which creates build configs and image streams for all of the required images.
+By default the buildconfigs are set up to pull the master branch of both this repository and the ManageIQ main repository.
 
-To execute new builds after the first (automatically started) you can execute:
+To deploy builds into the project `test` for a feature branch called `my_feature` which existed in both your fork of manageiq and manageiq-pods you would deploy the template like this:
 
 ```bash
-$ oc start-build -n <your-namespace> manageiq-pods
+$ oc process -f templates/helpers/build-template.yaml -p IMAGE_NAMESPACE=test -p SOURCE_REPOSITORY_URL=https://github.com/<your_fork_name>/manageiq-pods -p SOURCE_REPOSITORY_REF=my_feature -p MIQ_ORG=<your_fork_name> -p MIQ_REF=my_feature
```

Comment:
I thought you renamed MIQ to MANAGEIQ or did I misread that somewhere?

Reply:
Nah, we were talking about `IMG` => `IMAGE`, but I can change this too.